changelogs/unreleased/gh-8226-multiple-same-field-update.md

```diff
@@ -0,0 +1,3 @@
+## feature/box
+
+* Allow multiple update for the same field (gh-8226).
```

Comment:
Lets at least mention that you mean tuple fields, not just some other fields. Also you need to write a docbot request and go through the commit messages to fix typos, add ticket references.